Engine/AI/NavMesh.cpp

```diff
@@ -766,7 +773,7 @@ void NavMesh::GetVerticesScene()
 
 	for (auto mesh : App->renderer->meshes)
 	{
-		for (int i = 0; i < mesh->mesh_to_render.get()->vertices.size(); ++i)
+		for (size_t i = 0; i < mesh->mesh_to_render.get()->vertices.size(); ++i)
```

Comment:
Why here you changed this from int to size_t and in line 660 you did the reverse thing? Changing from size_t to int.

Reply:
Just to avoid warnings in the for comparison